### PR TITLE
farsi translation of properties

### PR DESCRIPTION
**Pull request by esfand-r**
@ehsanrohani @mvalipour please review when you had a chance

### CODE REVIEW

File: conf/messages.fa [hunk 1]
```diff
@@ -3,50 +3,50 @@ title = بیانیه واكنشگر
...
-sign.hello = Hello
-sign.addYourSignature = Add your signature
-sign.thankYouAndSpreadTheWord = Thank you! And spread the word!
-sign.removeYourSignature = Remove your signature
-sign.hiThere = Hi there,{0} You need to be logged to sign the manifesto
-sign.signInWith = Sign In with {0}
+sign.hello = درود
+sign.addYourSignature = امضای خود را اضافه کنید
+sign.thankYouAndSpreadTheWord = با تشکر! به دیگران اطلاع رسانی کنید!
+sign.removeYourSignature = امضای خود را پاک کنید
```
Comment (mvalipour) [inline conf/messages.fa]: `حذف` instead of `پاک` ?

**Comment by esfand-r** [comment]
@rasummer as promised. Cheers

File: conf/messages.fa (hunk 1 above)
Comment (esfand-r) [inline conf/messages.fa]: Thanks. done

**Comment by esfand-r** [comment]
Thanks @mvalipour

**Pull request merged**